subdomains/blog/_posts/2020-12-21-announcing-volta-100.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Announcing Volta 1.0.0"
+date:   2020-12-16 08:00:00 -0700
+categories: update
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+The Volta team is thrilled to announce that Volta 1.0.0 is now available! **Volta is a fast, reliable manager for JavaScript tools that takes the hassle out of switching versions**, allowing you to focus on your work, not your tools.
+<!--more-->
+
+# What does 1.0.0 mean for Volta?
+
+The 1.0.0 release of Volta means more than, "we implemented some high-impact functionality and finally closed some long-standing issues." From the beginning, Volta was built to empower engineers and engineering teams to build amazing JS applications without having to worry about their environment or tooling. For individual engineers, Volta unobtrusively makes sure your tooling is aligned with the needs of your project, allowing you to stay focused on your work. For engineering teams, Volta allows you to manage your infrastructure decision-making centrally without slowing down individual team members. When the team decides to change versions of Node, for example, that doesn't require explicit effort on the part of each developer. With version 1.0.0, we believe Volta delivers on the promise of making tool management disappear.
```

Comment:
This says what 1.0 doesn't mean but it doesn't _quite_ make clear what it does mean. The last sentence of the paragraph comes closest but (a) doesn't really explain why we believe it, and (b) buries the lede.

Some suggestions:
- Can we articulate why we believe 1.0 delivers on the promise? (My guess: we closed the final remaining gaps (link and upgrade) to make all standard tools work transparently under Volta control.)
- I feel like we're missing an opportunity to really clearly make this the moment everyone should jump on board. Maybe there's a second paragraph or subsection here that says something like "Volta is ready for general use!" and talks about how not only is the core vision fulfilled, but we're also making a commitment to stability? We could even talk about how Volta has changed less and less since 0.5 or 0.6 or whatever, and that with 1.0 we feel confident enough in the design that we will commit to not breaking people's workflows or scripts.

Reply:
And one additional thought: ideally there would be a "why should I use a JS tool manager?" story for people who've never even used nvm et al. If we get lucky enough to go viral, a bunch of people will be seeing Volta for the first time with this post. We want to make sure they don't get confused or bored before they find out why they should be interested.

Maybe before the "What does 1.0.0 mean" section, we should have a super short "Why should I use a tool manager?" section that starts with something like, "If you've never used a tool manager before…"